tutorials/telco/charmed-osm-get-started/charmed-osm-get-started.md

```diff
@@ -0,0 +1,840 @@
+mariadb-k8s/0*     active    idle   10.1.50.16  3306/TCP                    ready
+mon-k8s/0*         active    idle   10.1.50.31  8000/TCP                    ready
+mongodb-k8s/0*     active    idle   10.1.50.22  27017/TCP                   ready
+nbi-k8s/0*         active    idle   10.1.50.32  9999/TCP                    ready
+pol-k8s/0*         active    idle   10.1.50.29  80/TCP                      ready
+prometheus-k8s/0*  active    idle   10.1.50.23  9090/TCP                    ready
+ro-k8s/0*          active    idle   10.1.50.27  9090/TCP                    ready
+ui-k8s/0*          active    idle   10.1.50.33  80/TCP                      ready
+zookeeper-k8s/0*   active    idle   10.1.50.26  2181/TCP,2888/TCP,3888/TCP  ready
+```
+
+You can then export the `OSM_HOSTNAME` environmental variable:
+
+```bash
+$ export OSM_HOSTNAME=`juju status nbi-k8s | grep kubernetes | awk '{print $8}'`
+$ echo "export OSM_HOSTNAME=$OSM_HOSTNAME" >> ~/.bashrc
+```
+
+At this point, you can start interacting with your Charmed OSM installation by using the `osmclient.osm` command, for example:
+
+```bash
+$ osmclient.osm ns-list
++------------------+----+--------------------+---------------+-----------------+
+| ns instance name | id | operational status | config status | detailed status |
++------------------+----+--------------------+---------------+-----------------+
++------------------+----+--------------------+---------------+-----------------+
+```
+
+You can also access Charmed OSM web GUI at the IP address returned by the following command:
+
+```bash
+$ juju status ui-k8s | grep kubernetes | awk '{print $8}'
+10.152.183.236
+```
+
+### Add MicroStack as a VIM
+
+In order to add MicroStack as a VIM for Charmed OSM, execute the following command:
+```bash
+$ osmclient.osm vim-create --name microstack \
+      --user admin \
+      --password keystone \
+      --auth_url http://10.20.20.1:5000/v3 \
+      --tenant admin \
+      --account_type openstack \
+      --config='{security_groups: default,
+                 keypair: microstack,
+                 project_name: admin,
+                 user_domain_name: default,
+                 region_name: microstack,
+                 insecure: True,
+                 availability_zone: nova,
+                 version: 3,
+                 use_floating_ip: true}'
+c59b6823-65e5-4c64-9a36-86cea30ba1db
+```
+
+That’s it! You can start onboarding your VNFs.
+
+## Onboarding VNFs
+
+Duration: 15:00
+
+At this point you have MicroK8s installed, Charmed OSM deployed on top of MicroK8s and MicroStack configured as a VIM. We will now deploy a basic network service to exercise the environment.
+
+### Network Service
+
+Download sample NSD (Network Service Descriptor) and VNFD (Virtual Network Function Descriptor):
+
+```bash
+$ wget http://bit.ly/basic_vnfd -O hackfest_basic_vnf.tar.gz
+--2019-11-12 16:48:16--  http://bit.ly/basic_vnfd
```

Comment:
Output not needed, you should just put the two wget commands one after the other.